tilelang/tools/lower_trace/core.py

```
@@ -0,0 +1,1202 @@
+                # was compiled from TIR and is consumed downstream via
+                # host_mod.import_module().  A pure-Python proxy cannot be
+                # used here (the FFI boundary requires a real Module handle),
+                # so the unpatched module is returned.  Only the trace/display
+                # text reflects the user's edits.
+                if patched_text.rstrip() != codegen_text.rstrip():
+                    target_kind = ""
+                    try:
+                        _t = args[1] if len(args) > 1 else kwargs.get("target", "")
+                        target_kind = _t.kind.name
+                    except Exception:
+                        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)